### ISSUE TITLE
install problem raspberry pi

### ISSUE THREAD

**Issue by ceeslansb** [StuffNL/domoticz-woonveilig#4]
I have installed the module on a raspberry pi and tried it also on an ubuntu (virtual) desktop
On both machines i have the same failure in the log of domoticz
What is the problem?
Thanks in advance for your reply
Cees

Log file domoticz:
2017-12-01 16:42:27.476 Error: (Woonveilig) failed to load 'plugin.py', Python Path used was '/home/pi/domoticz/plugins/domoticz-woonveilig/:/usr/lib/python35.zip:/usr/lib/python3.5:/usr/lib/python3.5/plat-arm-linux-gnueabihf:/usr/lib/python3.5/lib-dynload'.
2017-12-01 16:42:27.476 Error: (woonveilig) Module Import failed, exception: 'ImportError'
2017-12-01 16:42:27.476 Error: (woonveilig) Module Import failed: ' Name: demjson'

on ubuntu
2017-12-02 13:20:59.943 Error: (Woonveilig) failed to load 'plugin.py', Python Path used was '/home/parallels/domoticz/plugins/domoticz-woonveilig/:/usr/lib/python35.zip:/usr/lib/python3.5/:/usr/lib/python3.5/plat-x86_64-linux-gnu:/usr/lib/python3.5/lib-dynload'. 
2017-12-02 13:20:59.943 Error: (woonveilig) Module Import failed, exception: 'ImportError' 
2017-12-02 13:20:59.943 Error: (woonveilig) Module Import failed: ' Name: demjson' 
2017-12-02 13:20:59.943 Error: (woonveilig) Error Line details not available.

**Comment by svdbeemt** [comment]
Hi, you are missing the demjson module. Did you follow the instructions on https://github.com/StuffNL/domoticz-woonveilig ?

**Comment by ceeslansb** [comment]
Hi Steven,
I did follow your instructions, but probably i have done something wrong.

I have figured out that the demjson module was in the python2.8 directory and not updated to python 3.
I have corrected the problem by deleting all the demjson stuff and repeating your instructions
After this the demjson file was in the python3.5 directory.
Your script has an append.sys command with python3.4 , I have changed this in 3.5
Now the module is working, I see devices and have only to figure out why  “last seen” is not updated.
"Last Seen”  stays on the initialization time
But in the log file I can see that the module is getting data, 
So far so good,
Kind regards
Cees
